wandb/env.py

```diff
@@ -36,6 +36,7 @@
 PROGRAM = "WANDB_PROGRAM"
 ARGS = "WANDB_ARGS"
 MODE = "WANDB_MODE"
+START_METHOD = "WANDB_START_METHOD"
```

Comment:
not needed

Reply:
I figured it would be good to keep all environment variables in one spot.  We have a real documentation crisis with the various settings that can be used...  The real fix is likely better docs in `wandb.Settings`.

Reply:
ok